lib/vkformat_check.c

```diff
@@ -15,72 +15,71 @@
 #include "vkformat_enum.h"
 
 bool
-isProhibitedFormat(VkFormat format)
-{
+isProhibitedFormat(VkFormat format) {
```

Comment:
Has the generator been modified to match the clang format or has clang modified this file? I don't see anything to tell clang to ignore it.

Reply:
clang-format modified it.

For generated files we really have various options:

  1. Not format them at all (would require moving generated files into a dedicated folder marked to not format)
  2. Format them (by explicitly calling e.g. `git clang-format` before your commit) - kind of the current proposal
  3. Integrate calling clang-format into the generation process

Reply:
I thought you were going to modify the generators to add a line that tells clang_format not to touch the file and to output in whatever format we eventually settle on.

How does clang-format get installed on one's system? My concern is for when we eventually move to generating the VkFormat related files from vk.xml in the Vulkan-Docs repo. Will #3 require installing clang-format in that repo.

Reply:
So it is an option to not auto-format the generated files, but then they also need to be moved to a dedicated directory, because clang-format styles can only be changed (in this case disabled) per directory.

It's up to us whether we want to do that instead, but with this prototype I proposed that it may be best to just format them as other files (and after they are regenerated they are also reformatted using `git clang-format`, per normal).

Reply:
> but then they also need to be moved to a dedicated directory, because clang-format styles can only be changed (in this case disabled) per directory.

What happened to this?

> I still have to update the auto-generated files and their scripts to include /* clang-format off */ in order to avoid auto-formatting them,

Reply:
Oh you're right. I completely forgot that's what we agreed on with @lexaknyazev.